metals/src/main/scala/scala/meta/internal/metals/codeactions/ImportMissingSymbol.scala

```diff
@@ -100,8 +100,7 @@ class ImportMissingSymbol(compilers: Compilers, buildTargets: BuildTargets)
         .flatten
         .toList
 
-      if (uniqueCodeActions.length > 1) {
-
+      if (codeActions.length > 1 && uniqueCodeActions.length > 0) {
```

Comment:
Why wouldn't `codeActions.length > 1` be enough here?

Reply:
Ach, I changed that and then remembered the issue. We actually do need both:
- `codeActions.length > 1` if it's just one action we don't want to show "Import all ..." since that doesn't make sense
- `uniqueCodeActions.length > 0` we don't want to show it if there are no unambiguous import to import, since the action would do nothing.